vars/joySlackNotifications.groovy

```diff
@@ -8,14 +8,14 @@
 
 
 /**
- * Notify the jenkins mattermost channel, only on master/release branches, and
+ * Notify the jenkins Slack channel, only on master/release branches, and
  * don't neglect colors or emoji.
 */
 void call(Map args = [:]) {
     String channel = args.channel ?: 'jenkins';
     String comment = args.comment ?: '';
 
-    // addapted from https://github.com/jenkinsci/mattermost-plugin/blob/mattermost-2.7.0/src/main/java/jenkins/plugins/mattermost/ActiveNotifier.java
+    // addapted from https://github.com/jenkinsci/slack-plugin/blob/slack-2.48/src/main/java/jenkins/plugins/slack/ActiveNotifier.java
```

Comment:
"adapted"

Reply:
Ha. That’s just what was there before, but I’ll fix that too :-)